source/applicability/transport_of_data.rst

```diff
@@ -41,6 +41,10 @@ RSMP connections can be established:
 Multiple supervisors
 ^^^^^^^^^^^^^^^^^^^^
 
+.. note::
+   Implementing support for multiple supervisors is not required unless
+   otherwise stated.
```

Comment:
stated in the sxl

Reply:
I think that it also could also be stated elsewhere, e.g. in the traffic signal requirements of a road authority.